scripts/nlp_language_modeling/merge_lora_weights/merge.py

```diff
@@ -195,8 +195,12 @@ def main(cfg) -> None:
     else:
         raise ValueError("need at least a nemo file or checkpoint dir")
 
+    if hasattr(model, 'mcore_gpt'):
+        assert (
+            model.mcore_gpt == False
```

Comment:
Could you please write this as `assert not model.mcore_gpt` for consistency?

I hope we'll introduce flake8 for static code analysis soon and this would be one of the places it complains about, see https://www.flake8rules.com/rules/E712.html.